src/Bindings/ManualBindings.cpp

```diff
@@ -3702,6 +3702,62 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cBookContent_GetPages(lua_State * tolua_S)
+{
+	// cBookContent::GetPages() -> table of strings
+
+	cLuaState L(tolua_S);
+	if (!L.CheckParamSelf("cBookContent"))
+	{
+		return 0;
+	}
+
+	cBookContent * BookContent = reinterpret_cast<cBookContent *>(tolua_tousertype(tolua_S, 1, nullptr));
+	L.Push(BookContent->GetPages());
+	return 1;
+}
+
+
+
+
+
+static int tolua_cBookContent_SetPages(lua_State * tolua_S)
+{
+	// cBookContent::SetPages(table)
+
+	cLuaState L(tolua_S);
+	if (
+		!L.CheckParamSelf("cBookContent") ||
+		!L.CheckParamTable(2))
+	{
+		return 0;
+	}
+
+	cBookContent * BookContent = reinterpret_cast<cBookContent *>(tolua_tousertype(tolua_S, 1, nullptr));
+
+	// Convert the input table into AStringVector:
+	AStringVector Pages;
+	int NumPages = luaL_getn(L, 2);
+	Pages.reserve(static_cast<size_t>(NumPages));
+	for (int i = 1; i <= NumPages; i++)
+	{
+		lua_rawgeti(L, 2, i);
+		AString Page;
+		L.GetStackValue(-1, Page);
+		if (!Page.empty())
+		{
+			Pages.push_back(Page);
+		}
+		lua_pop(L, 1);
```

Comment:
Looking at LuaChunkStay again, I would have to use `lua_rawgeti` then it should work. Another problem I just see is that I cannot add the string value to a vector that is outside of the function, all variables outside of the function are marked const.

Can I keep the current code?